src/scenes/home/landing/triGroup/triGroup.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LinkButton from 'shared/components/linkButton/linkButton';
+import blueMedalLogo from 'images/logos/small-blue-medal.png';
+import redMedalLogo from 'images/logos/small-red-medal.png';
+import styles from './triGroup.css';
+import staticData from './triGroup.json';
+
+const ClickableLogo = props => (
```

Comment:
can you move this within the definition of `TriGroup`? I like having PropTypes and DefaultTypes at the top of files for documentation.

Reply:
Okay